README.md

```diff
@@ -62,54 +111,3 @@ Several scripts form a pipeline for updating the database. Here is the data flow
                         └──────────────────────┘ │
                           └──────────────────────┘
 ```
-
-For running the entire data pipeline at once, do
-
-```bash
-$ npm install
-$ export GITHUB_CLIENT_ID=0123456789abcdef0123
-$ export GITHUB_CLIENT_SECRET=0123456789abcdef0123456789abcdef01234567
-$ export GITHUB_USERNAME=AurelienLourot
-$ export GITHUB_PASSWORD=********
-$ ./fetchAndCalculateAll.sh
-...
-data/
-  users/
-    262 users
-    largest: moul.json (20 KB)
-    total: 634 KB
-  contribs/
-    largest: moul.json (216 KB)
-    total: 3823 KB
-  repos/
-    8159 repos
-    largest: jlord/patchwork.json (379 KB)
-    total: 23889 KB
-  orgs.json: 639 KB
-  nonOrgs.json: 35 KB
-  total: 28984 KB
-
-=> 111 KB/user
-GitHub API key found.
-GitHub credentials found.
-
-real    78m44.200s
-user    2m58.520s
-sys     0m23.160s
-```
-
-> **NOTE**: API keys can be created [here](https://github.com/settings/developers).
-
-## Serving one more user
-
-```bash
-$ ./addUser.js newGreatUser
-$ ./fetchAndCalculateAll.sh
-```
-
-## Deleting a profile
-
-```bash
-$ ./rmUser.js formerUser "you asked us to remove your profile in https://github.com/ghuser-io/ghuser.io/issues/666"
-$ ./fetchAndCalculateAll.sh # will also perform some garbage collection
```

Comment:
This is quite interesting to state that garbage collection is happening. Repos and orgs get (re)moved every day.

Reply:
I thought to move it there https://github.com/ghuser-io/db/pull/1/files#diff-04c6e90faac2675aa89e2176d2eec7d8R65